### ISSUE TITLE
Move explanation of host MSI always copying files to default location into a referenceable doc

### ISSUE THREAD

**Issue by dagood** [dotnet/docs#19271]
This description is useful, but not easy to find or point people to. It should live in a Markdown file or something:

https://github.com/dotnet/core-setup/blob/e8cef8ea52a98092cc3c4fc252e611d88be29222/src/pkg/projects/Microsoft.NETCore.DotNetHost/host.wxs#L10-L21

**Comment by adegeo** [comment]
Should this be added to Windows Install Visual Studio section? The description isn't really clear about what the affect the user is experiencing that requires them to change something.

**Comment by dagood** [comment]
This was transferred here in triage, I don't know for sure where it would make sense to describe. I don't think this requires the user to change anything, is that a bar? I don't think the user *can* change anything about this.

Maybe a design doc would be (would have been) better? Even a issue describing the scenario would have been better IMO than a comment in some WiX source code, but it doesn't look like the change was tracked that way. 😕

The reason a user would be interested in this is if they notice that even though they installed VS to a secondary drive and the dotnet host got installed there, there's also an extra copy of the dotnet host installed on their main (C) drive. For example, they may be trying to save disk space on a small SSD, so look for an answer, even if the binary size is fairly small.

I originally filed this issue purely because I noticed this decision was documented pretty well in the WiX source but I couldn't track down any info about this decision in issues or docs. I believe I needed to link it to describe an issue to someone in a GitHub issue.

/cc @NikolaMilosavljevic
